be/src/connector/lake_connector.cpp

```diff
@@ -516,15 +520,33 @@ void LakeDataSource::init_counter(RuntimeState* state) {
     _total_columns_data_page_count =
             ADD_CHILD_COUNTER(_runtime_profile, "TotalColumnsDataPageCount", TUnit::UNIT, "SegmentRead");
 
+    // IO statistics
     // IOTime
     _io_timer = ADD_TIMER(_runtime_profile, "IOTime");
```

Comment:
Since you create a new timer called `IOTimeTotal`, so the timer `IOTime` can be removed? are they the same thing?